### PR TITLE
add homeBlog

### PR DESCRIPTION
**Pull request by Vjerem**

### CODE REVIEW

**Review by Mariion-62 — DISMISSED** [review]
It's okay

File: src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,14 @@
+import HomeBlog from './Components/HomeBlog';
+import WILDHEROES from './Images/WILDHEROES.png';
```
Comment (Hacene010) [inline src/App.jsx]: Why the Uppercase here??

Comment (Mariion-62) [reply src/App.jsx]: it's name of image, it's a problem?

File: src/Components/ArticleHomeBlog/index.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import './Index.css';
+
+const ArticleHomeBlog = ({ title, image }) => {
+  return (
+    <div className="newsArticles">
+      <h2>{title}</h2>
+      <img src={image} alt="images 1" />
```
Comment (Hacene010) [inline src/Components/ArticleHomeBlog/index.jsx]: On the alt, do you meen one image or a lot of images ? if it's one image plz remove the 's'

Comment (Mariion-62) [reply src/Components/ArticleHomeBlog/index.jsx]: when we try to change "images 1" --> "image" Eslint don't approuved. why?

**Comment by Vjerem** [comment]
delete branch

**Pull request closed without merging**